### PR TITLE
chore: add credofile

### LINKED ISSUE #34: Add credo file

### PR DESCRIPTION
**Pull request by Arkenan**
**Description**

A simple credo file describing which lint targets are run and which aren't.

Closes #34

### CODE REVIEW

File: .credo.exs [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+        #
+        # You can give explicit globs or simply directories.
+        # In the latter case `**/*.{ex,exs}` will be used.
+        #
+        included: [
+          "lib/",
+          "src/",
+          "test/",
+          "web/",
+          "apps/*/lib/",
```
Comment (mpaulucci) [inline .credo.exs:29]: maybe we just want to leave lib an test?

File: .credo.exs [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+          # in umbrella app is more clear to use module name to know from which app came
+          # {Credo.Check.Design.AliasUsage,
+          # [priority: :low, if_nested_deeper_than: 2, if_called_more_often_than: 0]},
+
+          # You can also customize the exit_status of each check.
+          # If you don't want TODO comments to cause `mix credo` to fail, just
+          # set this value to 0 (zero).
+          #
+          {Credo.Check.Design.TagTODO, [exit_status: 2]},
+          {Credo.Check.Design.TagFIXME, []},
```
Comment (MegaRedHand) [inline .credo.exs:95]: I think we should disable these checks.

**Comment by phklive** [comment]
How is this PR going guys ?

File: .credo.exs (hunk 1 above)
Comment (Arkenan) [reply .credo.exs]: Done

File: .credo.exs (hunk 2 above)
Comment (Arkenan) [reply .credo.exs]: Disabled.

**Review by MegaRedHand — APPROVED** [review]

**Pull request merged**